Infrastructure.Data/Context/WarrenEverestContext.cs

```diff
@@ -0,0 +1,25 @@
+﻿using DomainModels.Models;
+using Infrastructure.Data.Mapping;
+using Microsoft.EntityFrameworkCore;
+
+namespace Infrastructure.Data.Context
+{
+    public class WarrenEverestContext : DbContext
+    {
+        #region Props
+        public DbSet<Customer> Customer { get; set; }
+        #endregion
+
+        #region Required
+        protected override void OnModelCreating(ModelBuilder modelBuilder)
+        {
+            modelBuilder.ApplyConfigurationsFromAssembly(typeof(CustomerMapping).Assembly);
+        }
+        #endregion
+
+        public WarrenEverestContext(DbContextOptions<WarrenEverestContext> options) :base(options)
+        {
+        }
+
```

Comment:
EFCore faz DbSet automático
```suggestion
    {
        protected override void OnModelCreating(ModelBuilder modelBuilder)
        {
            modelBuilder.ApplyConfigurationsFromAssembly(typeof(CustomerMapping).Assembly);
        }

        public WarrenEverestContext(DbContextOptions<WarrenEverestContext> options) :base(options)
        {}
```